src/viewer/mobile/Panel.tsx

```diff
@@ -240,27 +259,57 @@ class Panel extends React.Component<Props & TwitchExtProps, State> {
 			return <Message dark={isDark}>Loading…</Message>;
 		}
 
-		const title = deck.name || "HSReplay.net";
-
 		return (
 			<TooltipProvider value={{ behaviour: TooltipBehaviour.FULLSCREEN }}>
 				<PanelDiv>
 					<Portal ref={ref => (this.portal = ref)} />
 					<PortalProvider value={{ portal: this.portal }}>
-						<Header>
-							<HeaderIcon src={HSReplayNetIcon} />
-							<h1>{title}</h1>
-							<CopyDeckButton deck={deck}>
-								{({ onClick, copied, disabled }: CopyDeckButtonChildProps) => (
-									<button onClick={onClick} disabled={disabled}>
-										{copied ? "Copied" : "Copy Deck"}
-									</button>
-								)}
-							</CopyDeckButton>
-						</Header>
-						<Scroller>
-							<CardList cardList={deck.cards} />
-						</Scroller>
+						{isBattlegrounds ? (
+							boardState &&
+							boardState.bobs_buddy_state != null &&
+							showBobsBuddy ? (
+								<BobsBuddy
```

Comment:
This needs some kind of background on dark mode, like LoadingScreen. Could also just be in PanelDiv above.